tests/evtimer_msg/Makefile

```diff
@@ -1,6 +1,7 @@
 include ../Makefile.tests_common
 
 USEMODULE += evtimer
+USEMODULE += xtimer
```

Comment:
This should not be needed. `xtimer` is (for now) still the default high level timer API an should be pulled in as dependency, unless the user explicitly wishes to do differently.

Reply:
Yes, right. But the test application makes direct use of `xtimer` methods. Thus, it has to be pulled in explicitly. Otherwise this test cannot be built when `evtimer` uses `ztimer_msec` as source.